scripts/check_gofmt.sh

```diff
@@ -4,5 +4,7 @@
 # Fail if a .go file hasn't been formatted with gofmt
 set -euo pipefail
 
-GO_FILES=$(find . -iname '*.go' -type f)   # All the .go files
-test -z $(gofmt -s -d "$GO_FILES" | tee /dev/stderr)
```

Comment:
This was broken already and was throwing `Argument list too long` error, eg:
https://cloud.drone.io/gomods/athens/2476/1/9

Reply:
Changes to these files are also to fix `gofmt` errors:

```
- e2etests/filesystem.go
- e2etests/run_athens.go
- pkg/observ/observ.go
- pkg/storage/s3/s3_test.go
```